tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+  "compilerOptions": {
```

Comment:
I'm not sure if these compiler options could be in conflict with the user's compiler options.

From the user's point of view, their production code will be compiled with their compilerOptions, while their test code will be compiled with Testy's compilerOptions